{
  "vulinfoid": "JVNDB-2020-015573",
  "vulinfodata": {
    "title": "OpenLDAP における到達可能なアサーションに関する脆弱性",
    "vulinfodescription": {
      "overview": "OpenLDAP には、到達可能なアサーションに関する脆弱性が存在します。"
    },
    "affected": {
      "affecteditem": [
        {
          "name": "Debian",
          "productname": "Debian GNU/Linux",
          "cpe": {
            "text": "cpe:/o:debian:debian_linux",
            "version": "2.2"
          }
        },
        {
          "name": "OpenLDAP Foundation",
          "productname": "OpenLDAP",
          "cpe": {
            "text": "cpe:/a:openldap:openldap",
            "version": "2.2"
          },
          "versionnumber": [
            "2.4.57 未満"
          ]
        },
        {
          "name": "アップル",
          "productname": "Apple Mac OS X",
          "cpe": {
            "text": "cpe:/o:apple:mac_os_x",
            "version": "2.2"
          }
        },
        {
          "name": "アップル",
          "productname": "macOS",
          "cpe": {
            "text": "cpe:/o:apple:macos",
            "version": "2.2"
          }
        }
      ]
    },
    "impact": {
      "cvss": [
        {
          "version": "2.0",
          "severity": {
            "text": "Medium",
            "type": "Base"
          },
          "base": "5",
          "vector": "AV:N/AC:L/Au:N/C:N/I:N/A:P"
        },
        {
          "version": "3.0",
          "severity": {
            "text": "High",
            "type": "Base"
          },
          "base": "7.5",
          "vector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H"
        }
      ],
      "impactitem": {
        "description": "サービス運用妨害 (DoS) 状態にされる可能性があります。"
      }
    },
    "solution": {
      "solutionitem": {
        "description": "ベンダより正式な対策が公開されています。ベンダ情報を参照して適切な対策を実施してください。"
      }
    },
    "related": {
      "relateditem": [
        {
          "type": "vendor",
          "name": "Apple Security Updates",
          "vulinfoid": "HT212529",
          "url": "https://support.apple.com/en-us/HT212529"
        },
        {
          "type": "vendor",
          "name": "Apple Security Updates",
          "vulinfoid": "HT212530",
          "url": "https://support.apple.com/en-us/HT212530"
        },
        {
          "type": "vendor",
          "name": "Apple Security Updates",
          "vulinfoid": "HT212531",
          "url": "https://support.apple.com/en-us/HT212531"
        },
        {
          "type": "vendor",
          "name": "Apple セキュリティアップデート",
          "vulinfoid": "HT212529",
          "url": "https://support.apple.com/ja-jp/HT212529"
        },
        {
          "type": "vendor",
          "name": "Apple セキュリティアップデート",
          "vulinfoid": "HT212530",
          "url": "https://support.apple.com/ja-jp/HT212530"
        },
        {
          "type": "vendor",
          "name": "Apple セキュリティアップデート",
          "vulinfoid": "HT212531",
          "url": "https://support.apple.com/ja-jp/HT212531"
        },
        {
          "type": "vendor",
          "name": "Debian Security Advisory",
          "vulinfoid": "DSA-4845-1",
          "url": "https://www.debian.org/security/2021/dsa-4845"
        },
        {
          "type": "vendor",
          "name": "debian-security-announce",
          "vulinfoid": "[SECURITY] [DLA 2544-1] openldap security update",
          "url": "https://lists.debian.org/debian-lts-announce/2021/02/msg00005.html"
        },
        {
          "type": "vendor",
          "name": "GitLab",
          "vulinfoid": "ITS#9423 ldap_X509dn2bv: check for invalid BER after RDN count",
          "url": "https://git.openldap.org/openldap/openldap/-/commit/8c1d96ee36ed98b32cd0e28b7069c7b8ea09d793"
        },
        {
          "type": "vendor",
          "name": "GitLab",
          "vulinfoid": "OPENLDAP_REL_ENG_2_4_57",
          "url": "https://git.openldap.org/openldap/openldap/-/tags/OPENLDAP_REL_ENG_2_4_57"
        },
        {
          "type": "vendor",
          "name": "OpenLDAP Bugs",
          "vulinfoid": "Issue 9423",
          "url": "https://bugs.openldap.org/show_bug.cgi?id=9423"
        },
        {
          "type": "advisory",
          "name": "Common Vulnerabilities and Exposures (CVE)",
          "vulinfoid": "CVE-2020-36230",
          "url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-36230"
        },
        {
          "type": "advisory",
          "name": "ICS-CERT ADVISORY",
          "vulinfoid": "ICSA-22-116-01",
          "url": "https://www.cisa.gov/uscert/ics/advisories/icsa-22-116-01"
        },
        {
          "type": "advisory",
          "name": "ICS-CERT ADVISORY",
          "vulinfoid": "ICSA-21-341-01",
          "url": "https://www.cisa.gov/uscert/ics/advisories/icsa-21-341-01"
        },
        {
          "type": "advisory",
          "name": "JVN",
          "vulinfoid": "JVNVU#91840304",
          "url": "https://jvn.jp/vu/JVNVU91840304/"
        },
        {
          "type": "advisory",
          "name": "JVN",
          "vulinfoid": "JVNVU#90035374",
          "url": "http://jvn.jp/vu/JVNVU90035374/index.html"
        },
        {
          "type": "advisory",
          "name": "National Vulnerability Database (NVD)",
          "vulinfoid": "CVE-2020-36230",
          "url": "https://nvd.nist.gov/vuln/detail/CVE-2020-36230"
        },
        {
          "type": "cwe",
          "name": "JVNDB",
          "vulinfoid": "CWE-617",
          "url": "http://cwe.mitre.org/data/definitions/617.html",
          "title": "到達可能なアサーション"
        }
      ]
    },
    "history": {
      "historyitem": [
        {
          "historyno": "1",
          "datetime": "2021-10-05T17:22:08+09:00",
          "description": "[2021年10月05日]\\n  掲載"
        },
        {
          "historyno": "2",
          "datetime": "2021-12-09T12:09:59+09:00",
          "description": "[2021年12月09日]\\n  参考情報：JVN (JVNVU#91840304) を追加\\n  参考情報：ICS-CERT ADVISORY (ICSA-21-341-01) を追加"
        },
        {
          "historyno": "3",
          "datetime": "2022-05-06T16:26:18+09:00",
          "description": "[2022年05月06日]\\n  参考情報：JVN (JVNVU#90035374) を追加\\n  参考情報：ICS-CERT ADVISORY (ICSA-22-116-01) を追加"
        }
      ]
    },
    "datefirstpublished": "2021-10-05T17:22:08+09:00",
    "datelastupdated": "2021-12-09T14:18:00+09:00",
    "datepublic": "2020-12-13T00:00:00+09:00"
  }
}
